hide/comp/Scene.hx

```diff
@@ -258,7 +258,7 @@ class Scene extends Component implements h3d.IDrawable {
 
 	function _loadTextureData( img : hxd.res.Image, onReady : Void -> Void, t : h3d.mat.Texture ) {
 		var path = ide.getPath(img.entry.path);
-		var img = new Element('<img src="file://$path" crossorigin="anonymous"/>');
+		var img = new Element('<img src="file://${ide.getUnCachedUrl(path)}" crossorigin="anonymous"/>');

```

Comment:
looks like you'll get `file://` twice here?

Reply:
Yes, thank you.

Fixed now on 6d061c6